README.md

```diff
@@ -22,6 +22,7 @@ $ npm install dynamo2es-lambda
 - **type** - { String } - Elasticsearch type to be used for all the documents; optional if `typeField` is provided
 - **[elasticsearch - alias: es]** - { Object } - Elasticsearch configuration; under the hood library uses [aws-elasticsearch-client][aws-elasticsearch-client-url]; for more information check [this documentation][aws-elasticsearch-client-url]
   - **[bulk]** - { Object } - aside from general Elasticsearch configuration, you can use this field to pass additional parameters to [bulk API][bulk-api-url]
+- **[elasticsearchVersion]** - { Number } - if >= 6, disables the requirement for a `type` or `typeField` option (mapping types are [deprecated in ES 6](https://www.elastic.co/guide/en/elasticsearch/reference/6.0/removal-of-types.html))
```

Comment:
`elasticsearch` config option takes an optional `apiVersion` parameter, so I'd prefer to use it instead of adding a new one. https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/16.x/configuration.html

Reply:
Yeah, that makes total sense. I've assumed the default to be `5` to preserve the current default behavior, but the current Elasticsearch default is `7` so this should probably be bumped and the tests updated accordingly in the next major version of dynamo2es-lambda.